docs/source/modin/supported/resampling_supported.rst

```diff
@@ -76,15 +76,13 @@ Computations / descriptive stats
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``min``                     | Y                               |                                                    |
 +-----------------------------+---------------------------------+----------------------------------------------------+
-| ``nunique``                 | N                               |                                                    |
+| ``nunique``                 | Y                               |                                                    |
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``ohlc``                    | N                               |                                                    |
 +-----------------------------+---------------------------------+----------------------------------------------------+
-| ``pad``                     | N                               |                                                    |
-+-----------------------------+---------------------------------+----------------------------------------------------+
 | ``prod``                    | N                               |                                                    |
 +-----------------------------+---------------------------------+----------------------------------------------------+
-| ``quantile``                | N                               |                                                    |
+| ``quantile``                | Y                               |                                                    |
```

Comment:
Can you also update src/snowflake/snowpark/modin/plugin/docstrings/resample.py::quantile()?

Reply:
Done!